src/types.ts

```diff
@@ -69,6 +82,7 @@ export interface TaskInstance {
   params: InstanceParams;
 }
 
+// FIXME: RewardType does not match with nft-server
```

Comment:
I think we should follow the nft-server's type? @woomurf could you confirm?

Reply:
It seems that only the name, description, and contractAddress of params need to be changed to string.  
Currently, there is no description of rewardType in the nft server, but it should be added. I will add it next PR.

Reply:
In the nft server, description is in the params.
```javascript
export type RewardType = {
  id: string,
  category: RewardTypeCategory,
  params: {
    name: string,
    description: string,
    contractAddress?: string,
    [key: string]: string | undefined,
  },
};
```
However, in ainftJs, the description is outside of the params.
```javascript
export interface RewardType {
  id: string;
  category: RewardTypeCategory;
  description: string;
  params: {
    name: true;
    description: true;
    contractAddress?: true;
  } & InstanceParams;
}
```
Where should we put the description? @woomurf

Reply:
Anyway, I will change the name, description, and contractAddress to string type, thanks!

Reply:
outside description means description of reward type.  and description in params means description of reward instance. So. description in params could be fill by event creator when create event. we need both.

Reply:
Thanks, then I will just change the name, description, and contractAddress to string type!

Reply:
Fixed, thanks!